source/Gamerules.cpp

```diff
@@ -79,6 +79,55 @@ void Gamerules::Load(const DataNode &node)
 
 
 
+string Gamerules::GetString(const string &rule) const
+{
+	if(rule == "disabled fighters avoid projectiles")
+	{
+		if(fighterHitPolicy == FighterDodgePolicy::ALL)
+			return "all";
+		if(fighterHitPolicy == FighterDodgePolicy::NONE)
+			return "none";
+		if(fighterHitPolicy == FighterDodgePolicy::ONLY_PLAYER)
+			return "only player";
+	}
+	return "";
```

Comment:
It probably gets optimized anyway, but I prefer this:
```suggestion
	return {};
```